WIP: Add podcast feature to website (#992)
bitcoinops/bitcoinops.github.io: _posts/en/podcast/2023-02-23-newsletter-recap.md

---
title: 'Bitcoin Optech Newsletter #239 Recap Podcast'
permalink: /en/podcast/2023/02/23/
name: 2023-02-23-recap
slug: 2023-02-23-recap
type: podcast
layout: podcast-episode
lang: en
---

> Join Mark "Murch" Erhardt and Mike Schmidt as they are joined by James O'Beirne,
> Christian Decker, and Russell O'Connor to discuss [Newsletter #239][news239].

Reviewer: The “_Join_ […] as they are _joined_ by […]” kinda bugs me. Maybe:

```suggestion
Join Mark "Murch" Erhardt and Mike Schmidt as they discuss [Newsletter #239][news239] with James O'Beirne,
Christian Decker, and Russell O'Connor.
```

```suggestion
Mark "Murch" Erhardt and Mike Schmidt are joined by James O'Beirne,
Christian Decker, and Russell O'Connor to discuss [Newsletter #239][news239].
```
